TestConsoleApp/Program.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
```

Comment:
Oh! Didn't know traces of this were in the program.cs file. I initially tried creating a console app to call the method, but deleted it because I couldn't get it to do so. Thought it was all gone. Will fix it now.